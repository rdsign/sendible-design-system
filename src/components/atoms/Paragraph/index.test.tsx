import { create } from 'react-test-renderer';
import { Paragraph } from '.';
import { Template } from '../../../../spec/template';

describe('Paragraph', () => {
    const component = (
        <Template>
            <Paragraph size="s20">
                Sendible<i> Rocks!</i>
            </Paragraph>
        </Template>
    );

    test('should match snapshot', () => {
        const snapshot = create(component);
        expect(snapshot).toMatchSnapshot();
    });
});
