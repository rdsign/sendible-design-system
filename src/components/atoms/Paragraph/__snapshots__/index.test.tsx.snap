// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should match snapshot 1`] = `
.c0 {
  color: #201E24;
  font-size: 2rem;
  font-weight: 400;
  line-height: 1.5em;
}

.c0 i,
.c0 a {
  color: #4D36FF;
}

<p
  className="c0"
  size="s20"
>
  Sendible
  <i>
     Rocks!
  </i>
</p>
`;
